dev/com.ibm.ws.jbatch.cdi.2.1/src/com/ibm/ws/jbatch/cdi/internal/CDIBatchArtifactFactoryImpl.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright 2012 International Business Machines Corp.
```

Comment:
```suggestion
 * Copyright 2012, 2023 International Business Machines Corp.
```
Should this be EPL 2.0 license?